<ui version="4.0" >
 <class>IconsOptionsWidgetClass</class>
 <widget class="QWidget" name="IconsOptionsWidgetClass" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>472</width>
    <height>307</height>
   </rect>
  </property>
  <property name="sizePolicy" >
   <sizepolicy vsizetype="Expanding" hsizetype="Expanding" >
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle" >
   <string>IconsOptionsWidget</string>
  </property>
  <layout class="QVBoxLayout" >
   <property name="leftMargin" >
    <number>0</number>
   </property>
   <property name="topMargin" >
    <number>0</number>
   </property>
   <property name="rightMargin" >
    <number>0</number>
   </property>
   <property name="bottomMargin" >
    <number>0</number>
   </property>
   <item>
    <widget class="QTabWidget" name="tabWidget" >
     <property name="currentIndex" >
      <number>0</number>
     </property>
     <widget class="QWidget" name="tabDefault" >
      <attribute name="title" >
       <string>Default</string>
      </attribute>
      <layout class="QVBoxLayout" >
       <item>
        <widget class="QLabel" name="label" >
         <property name="text" >
          <string>Default iconset:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListWidget" name="lwtDefaultIconset" >
         <property name="verticalScrollMode" >
          <enum>QAbstractItemView::ScrollPerPixel</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabService" >
      <attribute name="title" >
       <string>Services</string>
      </attribute>
      <layout class="QVBoxLayout" >
       <item>
        <widget class="QLabel" name="label_2" >
         <property name="text" >
          <string>Service specific icons (expresion performed on domane):</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTableWidget" name="twtServiceRules" >
         <property name="selectionMode" >
          <enum>QAbstractItemView::SingleSelection</enum>
         </property>
         <property name="selectionBehavior" >
          <enum>QAbstractItemView::SelectRows</enum>
         </property>
         <property name="columnCount" >
          <number>2</number>
         </property>
         <column>
          <property name="text" >
           <string>RegExp</string>
          </property>
         </column>
         <column>
          <property name="text" >
           <string>Iconset</string>
          </property>
         </column>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" >
         <item>
          <spacer>
           <property name="orientation" >
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" >
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="pbtAddService" >
           <property name="text" >
            <string>Add</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pbtDeleteService" >
           <property name="text" >
            <string>Delete</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabContact" >
      <attribute name="title" >
       <string>Contacts</string>
      </attribute>
      <layout class="QVBoxLayout" >
       <item>
        <widget class="QLabel" name="label_3" >
         <property name="text" >
          <string>Contact specific icons (expresion performed on bare JID):</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTableWidget" name="twtContactRules" >
         <property name="selectionMode" >
          <enum>QAbstractItemView::SingleSelection</enum>
         </property>
         <property name="selectionBehavior" >
          <enum>QAbstractItemView::SelectRows</enum>
         </property>
         <property name="columnCount" >
          <number>2</number>
         </property>
         <column>
          <property name="text" >
           <string>RegExp</string>
          </property>
         </column>
         <column>
          <property name="text" >
           <string>Iconset</string>
          </property>
         </column>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" >
         <item>
          <spacer>
           <property name="orientation" >
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" >
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="pbtAddContact" >
           <property name="text" >
            <string>Add</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pbtDeleteContact" >
           <property name="text" >
            <string>Delete</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabCustom" >
      <attribute name="title" >
       <string>Custom</string>
      </attribute>
      <layout class="QVBoxLayout" >
       <item>
        <widget class="QLabel" name="label_4" >
         <property name="text" >
          <string>Custom icons (expresion performed on full JID):</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTableWidget" name="twtCustomRules" >
         <property name="selectionMode" >
          <enum>QAbstractItemView::SingleSelection</enum>
         </property>
         <property name="selectionBehavior" >
          <enum>QAbstractItemView::SelectRows</enum>
         </property>
         <property name="columnCount" >
          <number>2</number>
         </property>
         <column>
          <property name="text" >
           <string>RegExp</string>
          </property>
         </column>
         <column>
          <property name="text" >
           <string>Iconset</string>
          </property>
         </column>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" >
         <item>
          <spacer>
           <property name="orientation" >
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" >
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="pbtAddCustom" >
           <property name="text" >
            <string>Add</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pbtDeleteCustom" >
           <property name="text" >
            <string>Delete</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <resources/>
 <connections/>
</ui>
